test/unit/external_return_label_request_test.rb

```diff
@@ -118,24 +118,24 @@ def test_customer_address2
   end
 
   def test_sanitize
-    assert_equal @external_request_label_req.send(:sanitize,'   ').length, 0
-    assert_equal @external_request_label_req.send(:sanitize, 'some string   '), 'some string'
-    assert_equal @external_request_label_req.send(:sanitize, {}), nil
-    assert_equal @external_request_label_req.send(:sanitize, nil), nil
-    assert_equal @external_request_label_req.send(:sanitize, nil), nil
-    assert_equal @external_request_label_req.send(:sanitize, []), nil
-    assert_equal @external_request_label_req.send(:sanitize, (1..100).to_a.join("_")).size, ExternalReturnLabelRequest::CAP_STRING_LEN
+    assert_equal 0, @external_request_label_req.send(:sanitize,'   ').length
```

Comment:
`assert_equal "", @external_request_label_req.send(:sanitize,'   ')` is definitely a better option here.